grc/satellites_snet_deframer.block.yml

```diff
@@ -9,9 +9,13 @@ parameters:
     options: ['True', 'False']
     option_labels: ['Yes', 'No']
 -   id: buggy_crc
-    label: Verbose
+    label: Buggy CRC
     dtype: bool
     default: True
+-   id: userdata_only
+    label: UserData Only
+    dtype: bool
+    options: ['True', 'False']
```

Comment:
What does this do? I don't see it used elsewhere.

Reply:
I was testing another thing, but yes, it should not be here.